Tests/test_SeqIO.py

```diff
@@ -672,18 +666,7 @@ def perform_test(
                         record = SeqIO.read(h, t_format, given_alpha)
                     self.assertIsInstance(base_alpha, given_base.__class__)
                     self.assertEqual(base_alpha, given_base)
-            for given_alpha in bad:
-                # These should all fail...
-                with open(t_filename, mode) as h:
-                    records6 = SeqIO.parse(h, t_format, given_alpha)
```

Comment:
I'm thinking that the SeqIO etc high level functions which take an alphabet will instead take the molecule type (perhaps under the same argument name), so this test would make sense with only minor edits.